practical-penguins/trivia_tavern/twilio_messenger/models.py

```diff
@@ -0,0 +1,43 @@
+from typing import Tuple, List, Optional
+
+from django.db import models
+from trivia_runner.models import Player
+
+
+class ScoreTracker(models.Model):
+    player_phone = models.CharField(max_length=12)
+    team_name = models.CharField(max_length=24, default='')
+    points = models.IntegerField(default=0)
+    session_code = models.CharField(max_length=6)
+    answered_this_round = models.BooleanField(default=False)
+
+    @staticmethod
+    def get_score_list(session_code: int) -> List[Optional[Tuple[str, int]]]:
+        """get_score_list is a class method that returns all players and their score_list
```

Comment:
It's not needed to specify the name and method type within the docstring, instead, it should really focus on what it does and how the passed args are used. PEP 257 goes over this. In this specific case, I would prefer to see something like "Return all players and their score_list for the specified *session_code*".